influxdb/influxdb08/dataframe_client.py

```diff
@@ -13,6 +13,16 @@
 from .client import InfluxDBClient
 
 
+def total_seconds(td):
```

Comment:
But it could be moved to influxdb/total_seconds.py?

This does not need to be a class method.